.github/workflows/dev-mode-layer-publish.yml

```diff
@@ -66,6 +66,71 @@ jobs:
           cd node
           ./scripts/publish-extension-layers.js \
             --bucket-name sls-layers-registry \
-            --layer-basename sls-external-extension \
+            --layer-basename sls-external-extension-x86 \
+            --version $VERSION \
+            --layer-filename ../go/packages/dev-mode/dist/extension.zip
+
+  publishNewVersion_arm64:
+    name: Publish new version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '>=1.18.0'
+      - name: Retrieve ~/.npm from cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: npm-v16-${{ runner.os }}-refs/heads/main-${{ hashFiles('node/**package*.json') }}
+          restore-keys: npm-v16-${{ runner.os }}-refs/heads/main-
+      - name: Retrieve node_modules from cache
+        id: cacheNodeModules
+        uses: actions/cache@v2
+        with:
+          path: node/node_modules
+          key: node-modules-v16-${{ runner.os }}-refs/heads/main-${{ hashFiles('node/package.json') }}
+          restore-keys: node-modules-v16-${{ runner.os }}-refs/heads/main-
+      - name: Install Node.js and npm
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+          registry-url: https://registry.npmjs.org
+      - name: Install main project dependencies
+        if: steps.cacheNodeModules.outputs.cache-hit != 'true'
+        run: |
+          cd node
+          npm update --save-dev --no-save
+      - name: Build layer for arm64
+        run: |
+          cd go/packages/dev-mode
+          make build_arm64
```

Comment:
Can't we add arm64 publish step into previous jobs? If I see correctly most of the steps are repeated, and we could just add arm64 steps there on top (?)

Reply:
Yeah we could do that 👍